components/UserDetails.tsx

```diff
@@ -46,11 +54,11 @@ const UserDetails = ({
         <Modal.Header closeButton>
           <Modal.Title>
             <User size={30} style={{ marginRight: 10 }} />
-            {user.email}
+            {user?.email || user.name}
           </Modal.Title>
         </Modal.Header>
         <Modal.Body>
-          {type === 'api' && (
+          {type === 'api' && 'apiKeys' in user && (
```

Comment:
I can't believe this is the first time I've seen the `in` syntax!

Reply:
It's really old and I think you're supposed to use `hasOwnProperty` now but that gave me typescript errors so it seems like this is the way to go